source/visionEnhancementProviders/NVDAHighlighter.py

```diff
@@ -297,7 +297,7 @@ def _updateEnabledState(self):
 			settings.highlightFocus,
 			settings.highlightNavigator,
 		]
-		if any(settingsToTriggerActivation):
+		if NVDAHighlighter.isEnabledInConfig() and any(settingsToTriggerActivation):

```

Comment:
I don't really understand this change. If it has to be already be enabled in config, then it can no longer be enabled by checking just one of the options.

Reply:
Ugh yes, you're right. I will fix this.
The change is there because there could be a situation where the three boxes are enabled, yet the highlighter isn't active because the provider isn't loaded.

Reply:
I have reverted this.